### PR TITLE
Split oauth and console apps

### PR DESCRIPTION
**Pull request by bafonins**
**Summary:**

This PR changes the folder structure of the `webui` package and splits the oauth and console apps. Current approach makes it difficult to maintain both apps and tune similar functionality when needed. For example, there absolutely no need to initialize all the reducers from console, when only using the oauth app. Also, the way to determine the current user is different, as well as the server responses to fetch user entity.

**Changes:**

* Move files `webui/views/console` to `webui/console/views`
* Move files `webui/views/oauth` to `webui/oauth/views`
* Move redux primitives to `webui/console/store` and `webiu/oauth/store`
* Same for `webui/api`
* Move `access-token` to `webui/console`. the oauth app does nothing with it and shouldnt in the future
* Separate the initialization logic, but keep using the shared `<Init />` component
* Replace `Object.assign` with the spread operator where possible
* Fix console login flow

Also, since I was going through all the codebase some extra fixes:
* Fix the registered state of the create account page
* Include logout action creators to the oauth app, so there is no need to explicitly redirect to `/oauth/login`, let the `WithAuth` component to take care of this.

### CODE REVIEW

**Comment by johanstokking** [comment]
Which issue is this closing?

**Comment by bafonins** [comment]
Closes https://github.com/TheThingsNetwork/lorawan-stack/issues/107

**Comment by bafonins** [comment]
see https://github.com/TheThingsNetwork/lorawan-stack/issues/107#issuecomment-461207732

File: pkg/webui/console/store/middleware/app.js [hunk 1]
```diff
@@ -12,48 +12,42 @@
...
-import api from '../api'
-import * as accessToken from '../lib/access-token'
+import * as app from '../actions/app'
+import api from '../../api'
+import * as accessToken from '../../lib/access-token'
 
-const consoleLogic = createLogic({
-  type: _console.INITIALIZE,
+const consoleAppLogic = createLogic({
+  type: app.INITIALIZE,
```
Comment (kschiffer) [inline pkg/webui/console/store/middleware/app.js]: Idk might this be confusing? Still `_console` was not the best naming either, so I guess it doesn't matter.

**Pull request merged**